charts/piraeus/charts/etcd/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+appVersion: 3.2.26
```

Comment:
`appVersion` should match up with the image tag.

Reply:
good catch :+1: